generator/src/main/java/com/scottlogic/deg/generator/Guice/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * classes used to configure Google Guice dependency injection.
+ */
+package com.scottlogic.deg.generator.Guice;
```

Comment:
What is this file for?

Reply:
I'm guessing this is simply to add a javadoc to the package? I'd prefer we didn't go to this level of documentation, and leave the code as self-documenting as possible.